### PR TITLE
Datagrid - Search and filter integration

### PR DESCRIPTION
**Pull request by neolitec**
Added search filter in private-database / archives pages.

UU-768

### CODE REVIEW

File: client/app/private-database/database/archive/dump/private-database-database-archive-dump.html [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
 
-    <button class="oui-button oui-button_link oui-button_icon-left" type="button"
-            data-ng-click="goToArchivesList()">
-        <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
-        <span data-i18n-static="common_back"></span>
-    </button>
-
-    <h2 data-ng-bind-html="tr('privateDatabase_tabs_dumps_change') + ' ' + bdd.name"></h2>
+    <oui-back-button
+        data-on-click="goToArchivesList()"
```
Comment (tremblaymath) [inline client/app/private-database/database/archive/dump/private-database-database-archive-dump.html]: You should replace it with href

File: client/app/private-database/database/archive/dump/private-database-database-archive-dump.html [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+        data-title="{{tr('privateDatabase_tabs_dumps_change') + ' ' + bdd.name}}">
+    </oui-back-button>
 
     <oui-datagrid data-rows="bdd.dumps">
-        <oui-column data-title="tr('privateDatabase_bdd_name')" data-property="databaseName" data-sortable></oui-column>
+        <oui-column data-title="tr('privateDatabase_bdd_name')" data-property="databaseName" data-sortable data-type="string" data-searchable></oui-column>
```
Comment (tremblaymath) [inline client/app/private-database/database/archive/dump/private-database-database-archive-dump.html]: We should add filters for databaseName,  creationDate and deletionDate... we will wait the final calendar release before prodding.

File: client/app/private-database/database/archive/list/private-database-database-archive-list.html [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
 
-    <button class="oui-button oui-button_link oui-button_icon-left" type="button"
-            data-ng-click="goToList()">
-        <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
-        <span data-i18n-static="common_back"></span>
-    </button>
-
-    <h2 data-ng-bind-html="tr('privateDatabase_tabs_dumps_archive_change', [nbDayToDelete])"></h2>
+    <oui-back-button
+        data-on-click="goToList()"
```
Comment (tremblaymath) [inline client/app/private-database/database/archive/list/private-database-database-archive-list.html]: same, use href attribute

File: client/app/private-database/database/archive/dump/private-database-database-archive-dump.html (hunk 2 above)
Comment (tremblaymath) [inline client/app/private-database/database/archive/dump/private-database-database-archive-dump.html]: one-time binding for translations

File: client/app/private-database/database/archive/dump/private-database-database-archive-dump.html (hunk 1 above)
Comment (neolitec) [reply client/app/private-database/database/archive/dump/private-database-database-archive-dump.html]: I can't with the existing code. There is no state associated with the action.

**Comment by jleveugle** [comment]
Quality check OK ?

I close, please re-open with the right label :)

**Comment by neolitec** [comment]
[Label don't close please]?

**Comment by antleblanc** [comment]
@neolitec Thanks! Could you please resolve conflicts?

**Comment by antleblanc** [comment]
ping @neolitec still has a conflicts.

**Comment by jleveugle** [comment]
Hello @neolitec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**